idd/Energy+.idd.in

```diff
@@ -73923,6 +73923,321 @@ HeatPump:PlantLoop:EIR:Heating,
         \note quadratic curve = a + b*PLR + c*PLR**2 is typical, other univariate curves may be used
         \note PLR = part load ratio (hot load/steady state capacity)
 
+HeatPump:AirToWater:FuelFired:Heating,
+  \memo The object defines a fuel-fired absorption heat pump based on equation-fit models.
+  \min-fields 10
```

Comment:
`min-fields` should at least include all required fields, so that would be 18. But given that all of the remaining fields are pertinent to the model, I would set it to 30 so that all fields are present in idf form.

Reply:
Done